test/int64_test.dart

```diff
@@ -729,7 +729,7 @@ void main() {
     expect(-(Int64.MIN_VALUE + Int64(1)), Int64.MAX_VALUE);
   });
 
-  test("", () {
+  test("misc", () {
```

Comment:
This could really be better. If these tests are a grab bag, it would be good to reorganize them into some meaningful groups. Personally, I tend to split out the error-handling tests separately.